chat_shell/chat_shell/tools/sandbox/client.py

```
@@ -270,6 +278,23 @@ async def execute(
+                "[SandboxClient] Sandbox disappeared before execution, "
+                "recreating and retrying once"
+            )
+            self._clear_cached_sandbox()
+            sandbox_id, recreate_error = await self.ensure_sandbox()
+            if recreate_error:
+                return SandboxExecutionResult(
+                    success=False,
+                    error=f"Failed to create sandbox: {recreate_error}",
+                    status="failed",
+                )
+            execution_id, subtask_id, error = await self._start_execution(
```

Ruff (0.15.15)

[warning] Unpacked variable `execution_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)